pkg/chains/signing.go

```diff
@@ -171,7 +172,7 @@ func (o *ObjectSigner) Sign(ctx context.Context, tektonObj objects.TektonObject)
 			}
 
 			// Now store those!
-			for _, backend := range signableType.StorageBackend(cfg).List() {
+			for _, backend := range sets.List[string](signableType.StorageBackend(cfg)) {
```

Comment:
Maybe use `signableType.StorageBackend(cfg).UnsortedList()` here? If so, let's also do it in `pkg/chains/verifier.go` and `test/test_utils.go`.

Reply:
Fair consideration... but that would be a (minor) functional change in that the current API returns a sorted string slice: https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets#String.List

IDK if anyone relies on the list being sorted. WDYT?

Reply:
It probably doesn't matter, but let's avoid the surprise :)